Formula/xorriso.rb

```diff
@@ -1,9 +1,9 @@
 class Xorriso < Formula
   desc "ISO9660+RR manipulation tool"
   homepage "https://www.gnu.org/software/xorriso/"
-  url "http://ftpmirror.gnu.org/xorriso/xorriso-1.4.0.tar.gz"
-  mirror "https://ftp.gnu.org/gnu/xorriso/xorriso-1.4.0.tar.gz"
-  sha256 "0bd1e085015b28c24f57697d6def2fe84517967dc417554c0c3ccf1685ed0e56"
+  url "https://www.gnu.org/software/xorriso/xorriso-1.4.2.tar.gz"
```

Comment:
Hmm. Kind of. GNU have a standing request that where the software is served by them to use mirrors, officially because it's closer & quicker for users but also partially I suspect to stop their servers getting utterly rammed:

> GNU software is available on ftp.gnu.org under the directory /gnu. Please try to use one of the many mirrors of our site listed below: the mirrors will give you faster response. Some mirrors also provide a copy of the rest of ftp.gnu.org; try the parent directory if you're interested. You can use the generic url http://ftpmirror.gnu.org to automatically choose a nearby and up-to-date mirror.

However, in this case, the homepage doesn't make any mention of the FTP mirrors (most GNU "software" pages do) and directs people to the `software` site directly. Part of me thinks this is consequently safe to change, part of me disagrees.